### ISSUE TITLE
Code definition/references/peek doesn't work with files that have Python association but  have no .py suffix

### ISSUE THREAD

**Issue by versemonger** [microsoft/pylance-release#803]
Issue Type: <b>Bug</b>

Either of Python or Pylance plugins which displays code definition/references/peek seems to work only when a file has `.py` suffix and doesn't work when it has Python association but no .py suffix.

Extension version: 2020.12.424452561
VS Code version: Code 1.52.1 (ea3859d4ba2f3e577a159bc91e3074c5d85c0523, 2020-12-16T16:30:02.420Z)
OS version: Darwin x64 19.6.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz (16 x 2300)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 2|
|Memory (System)|32.00GB (0.14GB free)|
|Process Argv|--crash-reporter-id d7b7b40c-8bbc-459b-9688-e7ee8b7d7acf|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
openlogontheside:30221877
python763:30178808
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
python504:30227505
vswsl492cf:30211402
wsl2promptcf:30224613
pythonvsdeb440:30237827
unusedpromptcf:30224611
folderexplorercf:30224615
openfilemenucf:30224648
pythonvsded773:30236629
vsjup459cf:30237188

```

</details>

**Comment by erictraut** [comment]
A fix for this was recently introduced into pyright (the type checker that pylance is built upon). The next release of pylance will pick up the fix.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.1.0, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202110-6-january-2021
